src/components/Bookstore.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Calculator.css';
+
+const Bookstore = () => (
+  <Switch>
+    <Route exact path="/">
+      <div id="Books" />
+    </Route>
+    <Route path="/categories">
+      <h1>Under Construction</h1>
+    </Route>
+  </Switch>
```

Comment:
- [ ] Kindly move these routes to the top level container `App.js`.

Reply:
- [ ] Also there should be two `<Link>`s in the navbar on the App.js container which leads to the booklist and categories locations. You can put the NavBar as a component.